Scoop/Controller/Home/HomeTableViewCell.swift

```diff
@@ -33,21 +33,30 @@ class HomeTableViewCell: UITableViewCell {
     
     private func setupContainerView() {
         containerView.clipsToBounds = true
-        containerView.layer.masksToBounds = true
+        containerView.layer.masksToBounds = false
         containerView.layer.cornerRadius = 10
-        containerView.layer.borderWidth = 2
-        containerView.layer.borderColor = UIColor.borderColor.cgColor
+        containerView.backgroundColor = .white
+        containerView.layer.borderColor = UIColor.white.cgColor
+        
+        // MARK: Shadow setup
+        containerView.layer.shadowColor = UIColor.black.cgColor
+        containerView.layer.shadowOpacity = 0.2
+        containerView.layer.shadowRadius = 5
+        containerView.layer.shadowOffset = CGSize(width: 2, height: 2)
+        containerView.layer.shouldRasterize = true
+        containerView.layer.rasterizationScale = UIScreen.main.scale
+        
```

Comment:
I made an extension to UIView that adds a shadow to any view, but idk if it works on a tableview cell. Maybe try once merged

Reply:
sgtm!